Tests/GoogleGenerativeAITests/GoogleGenerativeAITests.swift

```diff
@@ -16,43 +16,46 @@ import XCTest
 @testable import GoogleGenerativeAI
 
 final class GenerativeLanguageTests: XCTestCase {
-  let apiKey = "<INSERT-YOUR-API-KEY>"
-
-  func testSetup() {
-    let client = GenerativeLanguage(apiKey: apiKey)
+  var client: GenerativeLanguage!
+  
+  override func setUp() {
```

Comment:
Make sure lines have no trailing space (use CTRL + I to re-indent the code and fix whitespace issues).